src/Router.js

```diff
@@ -54,6 +54,14 @@ export const Router = () => (
         />
       }
     />
+    <Route
```

Comment:
Just a note, don't consider this as a necessary request for change: you can also experiment with `Outlet` available from react-router-dom: https://reactrouter.com/en/main/components/outlet. This allows to nest routes which could be applicable to the case of the /cves/:cveId route

Reply:
Very cool component, didn't know about it, I will certainly take a look.